chatops/test-hubot-st2-badconfig.bats

```diff
@@ -0,0 +1,78 @@
+
+load '../test_helpers/bats-support/load'
+load '../test_helpers/bats-assert/load'
+
+
+ST2_CHATOPS_ENV_FILE='/opt/stackstorm/chatops/st2chatops.env'
+cd /opt/stackstorm/chatops
+
+@test "SETUP: make sure ENV variables are unset in st2chatops.env to test edge cases" {
+	`sed -e '/ ST2_API_KEY=/s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_TOKEN= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
```

Comment:
Additional request to double check if this all really works.

I'm not 100% sure as I didn't ran it, but whitespace after `ST2_AUTH_TOKEN= ` makes me think there could be issues in similar `sed` rules.

Reply:
@armab You are right about whitespace. Will remove it.